sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/autorest.md

```diff
@@ -0,0 +1,40 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: HybridConnectivity
+namespace: Azure.ResourceManager.HybridConnectivity
+require: https://github.com/Azure/azure-rest-api-specs/blob/3c162c839b8fe17544d9a3be8383a835dd42eb28/specification/hybridconnectivity/resource-manager/readme.md
+tag: package-2021-10-06-preview
+output-folder: Generated/
+clear-output-folder: true
+skip-csproj: true
+ 
+rename-rules:
+  Os: OS
+  Ip: IP
+  Ips: IPs
+  ID: Id
+  IDs: Ids
+  VM: Vm
+  VMs: Vms
+  VMScaleSet: VmScaleSet
+  DNS: Dns
+  VPN: Vpn
+  NAT: Nat
+  WAN: Wan
+  Ipv4: IPv4
+  Ipv6: IPv6
+  Ipsec: IPsec
+  URI: Uri
+  TypePropertiesType: EndpointType
```

Comment:
We should not use this dictionary to transform names of property - could we change this to a directive? You can add `x-ms-client-name` to the corresponding property to do this

Reply:
This name `TypePropertiesType` is generated by codegen which is cased by flatten